lib/centurion/deploy_dsl.rb

```diff
@@ -180,13 +180,20 @@ def tls_paths_available?
     Centurion::DockerViaCli.tls_keys.all? { |key| fetch(key).present? }
   end
 
-  def build_tls_params
-    return {} unless fetch(:tlsverify)
-    {
-      tls: fetch(:tlsverify || tls_paths_available?),
-      tlscacert: fetch(:tlscacert),
-      tlscert: fetch(:tlscert),
-      tlskey: fetch(:tlskey)
-    }
+  def build_server_params
+    opts = {}
+    if fetch(:tlsverify)
+      opts[:tls] = fetch(:tlsverify || tls_paths_available?)
+      opts[:tlscacert] = fetch(:tlscacert)
+      opts[:tlscert] = fetch(:tlscert)
+      opts[:tlskey] = fetch(:tlskey)
+    end
+
+    if fetch(:ssh)
```

Comment:
Running `set(:ssh, 'false')` would probably evaluate true in this line. Maybe `if fetch(:ssh, false) == true`?